cumulusci/tasks/bulkdata/tests/test_snowfakery.py

```diff
@@ -783,12 +783,11 @@ def test_explicit_channel_declarations(self, mock_load_data, create_task):
                 "recipe": Path(__file__).parent
                 / "snowfakery/simple_snowfakery.recipe.yml",
                 "run_until_recipe_repeated": 15,
-                "recipe_options": {"xyzzy": "Nothing happens", "some_number": 42},
```

Comment:
This was cut and paste from elsewhere and didn't need to be tested over and over.